simulation-runner/config/nodes/pe201523-catalogzero.json

```diff
@@ -0,0 +1,6 @@
+{
+    "certname": "w3h244u9nwii0y5.delivery.puppetlabs.net",
+    "simulation_class": "com.puppetlabs.gatling.node_simulations.PE201523CatalogZero",
```

Comment:
I don't see this class in this PR - should this be one of the `WithFeeder` classes?